curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-many-elements-within-a-single-div-element.md

```diff
@@ -17,9 +17,9 @@ Just like any other non-self-closing element, you can open a `div` element with
 
 # --instructions--
 
-Nest your "Things cats love" and "Things cats hate" lists all within a single `div` element.
+Nest your "`Things cats love`" and "`Things cats hate`" lists all within a single `div` element.
 
-Hint: Try putting your opening `div` tag above your "Things cats love" `p` element and your closing `div` tag after your closing `ol` tag so that both of your lists are within one `div`.
+Hint: Try putting your opening `div` tag above your "`Things cats love`" `p` element and your closing `div` tag after your closing `ol` tag so that both of your lists are within one `div`.
```

Comment:
```suggestion
Hint: Try putting your opening `div` tag above your "Things cats love" `p` element and your closing `div` tag after your closing `ol` tag so that both of your lists are within one `div`.
```